### ISSUE TITLE
db_import error

### ISSUE THREAD

**Issue by hktalent** [rapid7/metasploit-framework#9921]
mac os 
```
$ python -V
Python 2.7.14 :: Anaconda custom (64-bit)
$ruby -v
ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin17]
```
metasploit v5.0.0-dev-59cd5e38db
```
db_import /Users/xxxx/safe/myhktools/1099.xml
[*] Importing 'Masscan XML' data
[-] Error while running command db_import: opts must include a valid :workspace

Call stack:
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/util/db_manager.rb:35:in `process_opts_workspace'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/core/db_manager/import/nmap.rb:19:in `import_nmap_xml'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/core/db_manager/import.rb:95:in `import'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/core/db_manager/import.rb:151:in `import_file'
/Users/xxxx/safe/top20/metasploit-framework/lib/metasploit/framework/data_service/proxy/core.rb:101:in `method_missing'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/ui/console/command_dispatcher/db.rb:1478:in `block (3 levels) in cmd_db_import'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/ui/console/command_dispatcher/db.rb:1471:in `each'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/ui/console/command_dispatcher/db.rb:1471:in `block (2 levels) in cmd_db_import'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/ui/console/command_dispatcher/db.rb:1465:in `each'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/ui/console/command_dispatcher/db.rb:1465:in `block in cmd_db_import'
/Users/xxxx/.rvm/gems/ruby-2.5.1@metasploit-framework/gems/activerecord-4.2.10/lib/active_record/connection_adapters/abstract/connection_pool.rb:292:in `with_connection'
/Users/xxxx/safe/top20/metasploit-framework/lib/msf/ui/console/command_dispatcher/db.rb:1460:in `cmd_db_import'
/Users/xxxx/safe/top20/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:548:in `run_command'
/Users/xxxx/safe/top20/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:510:in `block in run_single'
/Users/xxxx/safe/top20/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:504:in `each'
/Users/xxxx/safe/top20/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:504:in `run_single'
/Users/xxxx/safe/top20/metasploit-framework/lib/rex/ui/text/shell.rb:208:in `run'
/Users/xxxx/safe/top20/metasploit-framework/lib/metasploit/framework/command/console.rb:48:in `start'
/Users/xxxx/safe/top20/metasploit-framework/lib/metasploit/framework/command/base.rb:82:in `start'
./msfconsole:49:in `<main>'
msf5 auxiliary(scanner/smb/smb_ms17_010) > workspace -v

Workspaces
==========

current  name        hosts  services  vulns  creds  loots  notes
-------  ----        -----  --------  -----  -----  -----  -----
         default     247    1401      111    2      451    1160
         szswdx      240    919       13     0      0      1958
         zhengfu     66     27        0      0      2      142
         myhacker    89     413       0      49     0      269
         chengdujc   93     8         0      0      0      124
         zf          415    893       0      1      0      3079
         gszb        1418   12972     533    28     162    22205
         lanlan      6      6         1      0      0      6
         gsnbzj      820    1733      124    8      20     3430
         other       9      9         1      0      0      9
         lfyy        12     11        5      0      482    14
*        gs2018      1842   4754      252    14     1      2717

```

**Comment by hktalent** [comment]
@busterb 
git pull
to
metasploit v5.0.0-dev-08c1cd5909
the same error

**Comment by busterb** [comment]
Hi, this should be addressed as of #9925, thanks.
